README.md

```diff
@@ -65,7 +68,7 @@ This sample performs the specific action of migrating Streams from the version 1
 Within the `generate_adapter_upgrade_mappings` function, the specified `adapter_type` is checked against a list of tested and incompatible adapter types.
 - If an incompatible adapter type is encountered, the sample will exit with an error message. 
 - If the adapter type is neither tested nor known to be incompatible, a warning is logged and the script continues.
-    - This is could a case where a new adapter type is being used, we welcome feedback to be sent to `samples@osisoft.com`
+    - This is could a case where a new adapter type is being used, we welcome feedback to be sent to `samples@aveva.com`
```

Comment:
was this intentional? did our email really change?

Reply:
I thought it did but it looks like we are still using osisoft. I will reset this one and the others that were changed.